.gitignore

```diff
@@ -70,3 +70,4 @@ appsettings.shelby.json
 src/RoadRegistry.BackOffice.UI/.env.staging
 src/RoadRegistry.BackOffice.UI/.env.development
 src/RoadRegistry.BackOffice.UI/.env.test
+*.sln.startup.json
```

Comment:
what's this?

Reply:
It's used by the extension `SwitchStartupProject for VS 2022` to allow pre-configured multiple (or single) startup projects.
E.g. I have 1 called "Core" which starts backoffice-api, commandhost, eventhost, extracthost, so that you can go through the entire workflow in 1 go.